src/components/Mapper/Trainers/TrainersModal.jsx

```diff
@@ -33,13 +61,25 @@ const TrainersModal = ({
       >
         <CloseIcon />
       </IconButton>
-
+      <Button
```

Comment:
Everything looks good to me. Instead of having a button that stretches the full screen, could you replace it with an IconButton of `ContentPasteGo` from `@mui/icons-material` and add it to the title bar next to the Close Button?

Reply:
sure, i'll try that